### PR TITLE
Created Docker image for P2P

### PR DESCRIPTION
**Pull request by adrienne-deriv**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -107,5 +107,8 @@
         "typescript": "^5.2.2",
         "vite-plugin-html-config": "^1.0.11",
         "vite-plugin-sass": "^0.1.0"
+    },
+    "optionalDependencies": {
+        "@rollup/rollup-linux-x64-gnu": "4.9.5"
```
Comment (amam-deriv) [inline package.json:112]: [Q] why is this needed?

Comment (amam-deriv) [reply package.json:112]: may use FROM --platform=linux/xxxxx

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ENV HUSKY=0
+
+RUN git clone https://github.com/deriv-com/p2p.git .
+
+RUN npm install
+
+ENV NODE_ENV=production
+RUN npm run build
+
+FROM --platform=$BUILDPLATFORM node:lts-alpine3.20
```
Comment (adrienne-deriv) [inline Dockerfile:16]: Change to multi-stage build to use `node:lts` for production environment

**Review by amam-deriv — APPROVED** [review]

**Review by farrah-deriv — APPROVED** [review]
